File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string temp;
        for(auto ch:s){
            if(isalnum(ch)){
                temp+=tolower(ch);
            }
        }
        int left=0,right=temp.size()-1;
        if(right==left){
            return true;
        }
        while(left<=right){
            if(temp[left]!=temp[right]){
                return false;
            }left++;
            right--;
        }
        return true;

    }
};

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        rett=""
        for i in range(len(s)):
            if(s[i].isalnum()):
                rett=rett+s[i]
        temp=rett.lower()
        i=0
        j=len(rett)-1

        while(i<=j):
            if(temp[i]!=temp[j]):
                return False
            else:
                i+=1
                j-=1
        return True